x-pack/platform/plugins/shared/ml/public/application/data_frame_analytics/pages/analytics_exploration/components/analytics_detail_flyout/analytics_detail_flyout.tsx

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiFlyout,
+  EuiFlyoutBody,
+  EuiFlyoutHeader,
+  EuiLoadingSpinner,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useJobInfoFlyouts } from '../../../../../jobs/components/job_details_flyout';
+import { useGetAnalytics } from '../../../analytics_management/services/analytics_service';
+import type { AnalyticStatsBarStats } from '../../../../../components/stats_bar';
+import type { DataFrameAnalyticsListRow } from '../../../analytics_management/components/analytics_list/common';
+import { useMlLocator, useNavigateToPath } from '../../../../../contexts/kibana';
+import { ML_PAGES } from '../../../../../../locator';
+import { ExpandedRow } from '../../../analytics_management/components/analytics_list/expanded_row';
+
+export const AnalyticsDetailFlyout = () => {
+  const {
+    isDataFrameAnalyticsDetailsFlyoutOpen,
+    closeActiveFlyout,
+    setActiveJobId,
+    activeJobId: analyticsId,
+  } = useJobInfoFlyouts();
+  const [analytics, setAnalytics] = useState<DataFrameAnalyticsListRow[]>([]);
+  const [, setAnalyticsStats] = useState<AnalyticStatsBarStats | undefined>(undefined);
+  const [, setErrorMessage] = useState<any>(undefined);
+  const [, setIsInitialized] = useState(false);
+  const [, setJobsAwaitingNodeCount] = useState(0);
+  const blockRefresh = false;
+  const getAnalytics = useGetAnalytics(
+    setAnalytics,
+    setAnalyticsStats,
+    setErrorMessage,
+    setIsInitialized,
+    setJobsAwaitingNodeCount,
+    blockRefresh
+  );
+  const getAnalyticsCallback = useCallback(
+    (id: string | null) => getAnalytics(true, id),
+    [getAnalytics]
+  );
+
+  // Subscribe to the refresh observable to trigger reloading the analytics list.
+
+  useEffect(
+    function getAnalyticsDetailsOnChange() {
+      if (!analyticsId) {
+        return;
+      }
+      getAnalyticsCallback(analyticsId);
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [analyticsId]
+  );
+  const analytic = useMemo(
+    () => analytics.find((a) => a.id === analyticsId),
+    [analytics, analyticsId]
+  );
+
+  const locator = useMlLocator()!;
+  const navigateToPath = useNavigateToPath();
+
+  const redirectToAnalyticsList = useCallback(async () => {
```

Comment:
Can the ID of the job be passed to the DFA jobs list, in the same way you do for Anomaly Detection jobs? (If this isn't currently supported, I would leave the functionality as you have it here).

Reply:
Updated here [`466ddec` (#207141)](https://github.com/elastic/kibana/pull/207141/commits/466ddec987d94e12d4afef7df5b5f17fd4a85ea8)